modules/odr_caption/Taskfile.caption.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+
+# dotenv: ['.env', '{{.ENV}}/.env', '{{.HOME}}/.env']
```

Comment:
Is this comment for future work or should it be removed?

Reply:
removed